packages/desktop/scripts/create-build-dir.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs')
+
+const __SOURCE_DIRECTORY__ = 'public/build'
+const __DESTINATION_DIRECTORY__ = `${__SOURCE_DIRECTORY__}/build`
+
+if (!fs.existsSync(__DESTINATION_DIRECTORY__)) {
+    fs.mkdirSync(__DESTINATION_DIRECTORY__)
+}
+
+fs.readdirSync(__SOURCE_DIRECTORY__).forEach((file) => {
+    if (file.includes('.node')) {
+        fs.copyFileSync(`${__SOURCE_DIRECTORY__}/${file}`, `${__DESTINATION_DIRECTORY__}/${file}`)
```

Comment:
It might be better to do this with the [CopyWebpackPlugin](https://webpack.js.org/plugins/copy-webpack-plugin/) like we do for the assets and translations:

https://github.com/iotaledger/firefly/blob/ed6bce8a69542ffd2b9fcae61c88b6f0a35f5247/packages/desktop/webpack.config.js#L130-L147

Reply:
Good to know for the future!